scripts/telemctl

```diff
@@ -91,6 +91,10 @@ telem_restart() {
   telem_start
 }
 
+telem_journal_cli() {
+  telem_journal "$@"
```

Comment:
Where is this function defined?

Reply:
Ah, it's an installed binary. Never mind :-P

Reply:
```telem_journal``` is not a function is a small cli that is compiled from journal/cli.c and it should be installed along probes like hprobe.